en/docs/getting-started/basic-architecture.md

```diff
@@ -0,0 +1,72 @@
+# Basic Architecture and Key Components
+
+[![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)]({{base_path}}/assets/img/get_started/basic-architecture.png)
+
+## Management Plane
+
+The Management Plane is where API creation and API management takes place.  It consists of portals (Publisher, Developer Portal, Admin Portal) for users to create and manage APIs, implement rate limiting policies, monitor, and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The Management Plane includes API Analytics dashboards, displaying various business insights.
+
+### Components
+
+#### API Publisher
+
+[![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+
+WSO2 API Manager’s API Publisher is a state-of-the-art GUI based tool for API development and management. The GUI is designed for API creators to develop, document, secure, test, and version APIs with ease. It’s also able to cater to more API management-related tasks such as publishing APIs, monetizing APIs, and applying rate limiting policies.
+
+#### API Developer Portal
+
+[![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
+
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to, and consume APIs securely and easily.
+
+#### API Analytics
+
+[![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)]({{base_path}}/assets/img/get_started/analytics.png)
+
+Monitoring and analytics of APIs deployed on WSO2 API Manager are provided by the WSO2 API Manager Analytics component. This component includes different types of dashboards to provide more detailed different statistical views of your APIs. Furthermore, you can secure your system by configuring alerts and notifications on pre-determined events to identify any unusual behaviors in near real-time.  For more information, see [Working with Analytics]({{base_path}}/learn/analytics/configuring-apim-analytics).
+
+
+## Data Plane
+
+The Data Plane is where the created API is exposed to the public consumers and acts as the proxy for API calls. This also provides additional capabilities such as enforcing security, rate limiting etc. (policy enforcement point).
```

Comment:
Why do we say "(policy enforcement point)" here? It looks like it is part of the draft and needs to be removed.